File: frontend/src/delivery/model/voucher.ts
/** 与后端 `Voucher.scala` 对齐 */

import type { VoucherId } from './ids'

export interface Voucher {
  id: VoucherId
  title: string
  discountAmount: number
  minSpend: number
  expiresAt: string
  remainingCount: number
}

File: backend/shared/src/main/scala/delivery/model/Voucher.scala
package delivery.model

/** 与前端 `frontend/src/delivery/model/voucher.ts` 对齐 */

final case class Voucher(
    id: String,
    title: String,
    discountAmount: Double,
    minSpend: Double,
    expiresAt: String,
    remainingCount: Int
)
